internal/broker/amqp/consumer.go

```diff
@@ -104,13 +104,15 @@ func (c *consumer) Start(logger *log.Logger, handlers map[string]func([]byte) er
 				"status":       "failed",
 				"responseTime": duration,
 				"error":        fmt.Sprintf("%+v", err),
-			}).Errorf("[consumer] [FAILED] Failed to handle message: nacking and requeing: %v", err)
+			}).Errorf("[consumer] [FAILED] Failed to handle message: trigger error handling and acking: %v", err)
 			// TODO: remove comments to allow for redelivery. This will put events
 			// into the unacknowledged state
 
 			// err := msg.Nack(false, true) if err != nil {
```

Comment:
This can be removed with the new handling

Reply:
Shouldn't we log it somewhere though? Maybe in the error handler?

Reply:
Log what? I refer to the comments?